### PR TITLE
fix(lerobot_local): warn when a normalization pipeline is present but inert

### PR DESCRIPTION
**Pull request by cagataycali**
## Problem

`lerobot_local` already warns when a checkpoint ships **no** `policy_postprocessor.json` (actions emitted in raw space). It stays **silent** for a subtler, equally broken case: a normalization pipeline that *is* present and active but whose stats do not cover the declared `action` / `observation.state` keys.

LeRobot's `NormalizerProcessorStep` returns a tensor **unchanged** when its lookup key is absent from the loaded stats:

```python
# lerobot/processor/normalize_processor.py  (_apply_transform)
norm_mode = self.norm_map.get(feature_type, NormalizationMode.IDENTITY)
if norm_mode == NormalizationMode.IDENTITY or key not in self._tensor_stats:
    return tensor   # <-- silent passthrough
```

So a pipeline whose stats keys don't match the feature keys normalizes **nothing**: `observation.state` reaches the model raw and predicted actions reach the robot without unnormalization — while `has_postprocessor` stays `True`, so the existing guard in `policy.py` never fires.

### Canonical trigger: `lerobot/smolvla_base`

The pretraining base checkpoint ships its normalizer stats keyed by the training dataset, not by the canonical feature names:

```
policy_preprocessor_step_5_normalizer_processor.safetensors keys:
  so100.buffer.action.mean / .std
  so100-red.buffer.action.mean / .std
  so100-blue.buffer.action.mean / .std
  (no observation.state stats; no bare 'action' key)
```

Loaded via `lerobot_local` (`image_keys=[camera1,camera2,camera3]`, `strict_keys=True`), the bridge is active and `has_postprocessor=True`, yet direct calls confirm **both** transforms are passthrough:

```
STATE  norm passthrough? True   (observation.state absent from stats)
ACTION unnorm passthrough? True  (bare 'action' absent from stats)
```

The policy then receives un-normalized proprioception and its output is applied un-normalized. In a headless MuJoCo SO-101 rollout (seed=42, 110 steps @30Hz, cube at a reachable pose), the arm reaches a fixed pose and never engages the cube — **body-delta ground truth: cube `dxy=0.00000`, `dz=-0.016` (settle only), min gripper–cube ≈ 9 cm, no grasp** — with no diagnostic that normalization was inert.

smolvla_base inert-normalization rollout

## Fix

- Add `ProcessorBridge.inert_normalization_features()` — inspects the pre/post normalizer steps and returns `"<key> (<type>/<mode>)"` for every declared, non-IDENTITY feature whose lookup key is absent from the loaded stats (so it will passthrough). A `NormalizerProcessorStep` is checked for observation features; an `UnnormalizerProcessorStep` for the action — mirroring what each step actually applies, so a feature the step never touches is not falsely flagged. IDENTITY (visual) features are never flagged.
- Extend the load-time guard in `LerobotLocalPolicy`: when a postprocessor **is** present but `inert_normalization_features()` is non-empty, warn — matching the no-silent-passthrough intent of the existing missing-postprocessor warning, and pointing the user at fine-tuning (which writes proper stats) or `processor_overrides`.

**Diagnostic only — no change to action values.** A properly-keyed (fine-tuned) checkpoint whose stats use the canonical `action` / `observation.state` keys is not flagged.

Emitted warning for `smolvla_base`:

```
lerobot_local: lerobot/smolvla_base has an ACTIVE normalization pipeline but its
stats do not cover ['observation.state (STATE/MEAN_STD)', 'action (ACTION/MEAN_STD)']
-- those features are passed through UN-normalized ...
```

## Tests

`tests/policies/lerobot_local/test_inert_normalization_detection.py` builds real (network-free) LeRobot `NormalizerProcessorStep`/`UnnormalizerProcessorStep` pipelines:
- dataset-prefixed stats (`so100.buffer.action`) → state + action flagged inert (FAILS pre-fix: `AttributeError`, method absent);
- canonical `action`/`observation.state` stats → nothing flagged;
- no pipelines → empty.

Local gate: `ruff` + `ruff format` clean; `mypy` clean on both changed modules; `tests/policies/lerobot_local/` (`test_processor_real_pipeline`, `test_norm_stats_fallback`, `test_action_diagnostics`, `test_policy`, `test_configure_embodiment`, new test) — 245 passed.

## Changed files

- `strands_robots/policies/lerobot_local/processor.py` — `inert_normalization_features()`
- `strands_robots/policies/lerobot_local/policy.py` — extend load-time guard
- `tests/policies/lerobot_local/test_inert_normalization_detection.py` — regression test
- `CHANGELOG.md`

### CODE REVIEW

**Review by yinsong1986 — APPROVED** [review]
## Summary

This PR closes a silent-passthrough gap in `lerobot_local`. The existing load-time guard warns only when a checkpoint ships *no* postprocessor; this adds detection for the subtler case where a normalization pipeline IS present and active (`has_postprocessor=True`) but its stats are keyed such that LeRobot's `NormalizerProcessorStep`/`UnnormalizerProcessorStep` silently pass the `action`/`observation.state` tensors through un-(un)normalized. The new `ProcessorBridge.inert_normalization_features()` inspects each normalizer step's declared, non-IDENTITY features and reports those whose runtime lookup key (`key` for observations, the bare `action` constant for actions) is absent from the loaded stats; `LerobotLocalPolicy._load_model` warns when the set is non-empty. The change is diagnostic-only — no action values change.

I verified the core claim against the pinned LeRobot (0.5.1) `normalize_processor.py`: the passthrough condition (`norm_mode == IDENTITY or key not in self._tensor_stats`) and the action lookup via the `ACTION` constant match what the new method checks, and the per-step gating (flag observation features on `NormalizerProcessorStep`, only the action on `UnnormalizerProcessorStep`) correctly mirrors what each step applies at runtime, so a feature the step never touches is not falsely flagged. Reconstructing the method against real (network-free) LeRobot steps reproduces the PR's three test outcomes exactly (dataset-prefixed → state+action flagged, canonical → empty, no pipelines → empty).

## What's good

- Correctness anchored to the actual library semantics (`_tensor_stats` lookup, `ACTION` constant), not a guess.
- Load-time, warn-only, `list[str]` return — no crash path: `except ImportError` is narrow, and every attribute access is `getattr`-guarded or None-checked before `.value`.
- The `else` branch reads `bridge.inert_normalization_features()` only after `bridge is None` is excluded, so no NoneType access.
- User-facing warning strings are plain ASCII (AGENTS.md: no emojis in user-facing strings).
- Tight scope; regression tests exercise real LeRobot processor steps rather than mocks.

## Verification suggestions

Standard CI (`hatch run lint && hatch run test`) plus the new `tests/policies/lerobot_local/test_inert_normalization_detection.py` cover this. Optional spot-check: load `lerobot/smolvla_base` through `lerobot_local` and confirm the new warning fires while a fine-tuned checkpoint (canonical `action`/`observation.state` stats) stays quiet.
